.github/actions/setup-sentry/action.yml

```diff
@@ -167,34 +167,12 @@ runs:
           services="${services} chartcuterie"
         fi
 
-        sentry devservices up $services &
-
-        # TODO: Use devservices kafka. See https://github.com/getsentry/sentry/pull/20986#issuecomment-704510570
         if [ "$NEED_KAFKA" = "true" ]; then
-          # This is *not* the production version. Unclear reason as to why this was chosen
-          # https://github.com/getsentry/ops/blob/c823e62f930ecc6c97bb08898c71e49edc7232f6/cookbooks/getsentry/attributes/default.rb#L631
-          docker run \
-            --name sentry_zookeeper \
-            -d --network host \
```

Comment:
need to convert everything over to things like sentry_zookeeper

or... we could `--network host` everything but feels less good